modules/katello_content_view.py

```diff
@@ -164,7 +162,8 @@ def main():
     )
 
     if not HAS_NAILGUN_PACKAGE:
-        module.fail_json(msg="Missing required nailgun module (check docs or install with: pip install nailgun")
+        module.fail_json(msg="Missing required nailgun module (check docs or install with: "
+                         "pip install nailgun")
```

Comment:
Actually, let me add a new PR for travis and pep8 so I can go through and fix the pep8 issues that exist.